linux-systemd/distributed/minio.service

```diff
@@ -15,6 +15,8 @@ PermissionsStartOnly=true
 
 EnvironmentFile=-/etc/default/minio
 ExecStartPre=/bin/bash -c "[ -n \"${MINIO_VOLUMES}\" ] || echo \"Variable MINIO_VOLUMES not set in /etc/defaults/minio\""
+ExecStartPre=/bin/bash -c "[ -n \"${MINIO_ACCESS_KEY}\" ] || echo \"Variable MINIO_ACCESS_KEY not set in /etc/defaults/minio\""
+ExecStartPre=/bin/bash -c "[ -n \"${MINIO_SECRET_KEY}\" ] || echo \"Variable MINIO_SECRET_KEY not set in /etc/defaults/minio\""
```

Comment:
Only one entry is needed.

Reply:
you mean chain the conditions in one entry?

Reply:
No why are you writing ExecStartPre ?

Reply:
We should write it in one line..